pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -321,26 +327,30 @@ func TestExpirePod(t *testing.T) {
 		},
 	}
 
-	for _, tc := range tests {
-		t.Run(tc.name, func(t *testing.T) {
-			cache := newCache(tc.ttl, time.Second, nil)
+	for i, tt := range tests {
+		t.Run(fmt.Sprintf("case_%d", i), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(tt.name, func(t *testing.T) {
```

Reply:
Although I think we are using more `tc` these days.

Reply:
For the original PR, this test has been changed in PR https://github.com/kubernetes/kubernetes/pull/110954 and I didn't notice the change. 
Update to the latest master branch. Thanks for the catch!